integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiJmsRecovery.java

```diff
@@ -0,0 +1,739 @@
+                                    .runtimeEncryptionSecret(encryptionSecretName))
+                        .introspectorJobActiveDeadlineSeconds(300L)));
+    setPodAntiAffinity(domain);
+    logger.info("Create domain custom resource for domainUid {0} in namespace {1}",
+            domainUid, domNamespace);
+    boolean domCreated = assertDoesNotThrow(() -> createDomainCustomResource(domain),
+            String.format("Create domain custom resource failed with ApiException for %s in namespace %s",
+                    domainUid, domNamespace));
+    assertTrue(domCreated, String.format("Create domain custom resource failed with ApiException "
+                    + "for %s in namespace %s", domainUid, domNamespace));
+  }
+
+  private void checkPodNotCreated(String podName, String domainUid, String domNamespace) {
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for pod {0} to be not created in namespace {1} "
+                    + "(elapsed time {2}ms, remaining time {3}ms)",
+                podName,
+                domNamespace,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(assertDoesNotThrow(() -> podDoesNotExist(podName, domainUid, domNamespace),
+            String.format("podDoesNotExist failed with ApiException for %s in namespace in %s",
+                podName, domNamespace)));
+  }
+
+  /*
+   * Verify the server MBEAN configuration through rest API.
+   * @param managedServer name of the managed server
+   * @returns true if MBEAN is found otherwise false
+   **/
+  private boolean checkManagedServerConfiguration(String managedServer) {
+    ExecResult result = null;
+    int adminServiceNodePort
+        = getServiceNodePort(domainNamespace, getExternalServicePodName(adminServerPodName), "default");
+    checkCluster = new StringBuffer("status=$(curl --user weblogic:welcome1 ");
+    checkCluster.append("http://" + K8S_NODEPORT_HOST + ":" + adminServiceNodePort)
+          .append("/management/tenant-monitoring/servers/")
+          .append(managedServer)
+          .append(" --silent --show-error ")
+          .append(" -o /dev/null")
+          .append(" -w %{http_code});")
+          .append("echo ${status}");
+    logger.info("checkManagedServerConfiguration: curl command {0}", new String(checkCluster));
+    try {
+      result = exec(new String(checkCluster), true);
+    } catch (Exception ex) {
+      logger.info("Exception in checkManagedServerConfiguration() {0}", ex);
+      return false;
+    }
+    logger.info("checkManagedServerConfiguration: curl command returned {0}", result.toString());
+    if (result.stdout().equals("200")) {
+      return true;
+    } else {
+      return false;
+    }
+  }
+
+  // Crate a ConfigMap with a model file to add a new WebLogic cluster
+  private void createClusterConfigMap(String configMapName, String modelFile) {
```

Comment:
looks like this is not used, can be removed

Reply:
removed